asp_interactive_configuration/ooasp/config.py

```diff
@@ -380,22 +439,18 @@ def save_png(self,directory:str="./out"):
         """
         ctl = Control(['--warn=none'])
         fbs = []
-        if self.simplified_encodings:
-            path = resources.files(ooasp.encodings_simple).joinpath("viz_config.lp")
-            ctl.load(str(path))
-            path = resources.files(ooasp.encodings_simple).joinpath("ooasp_aux_kb.lp")
-            ctl.load(str(path))
-        else:
-            path = resources.files(ooasp.encodings).joinpath("viz_config.lp")
-            ctl.load(str(path))
-            path = resources.files(ooasp.encodings).joinpath("ooasp_aux_kb.lp")
-            ctl.load(str(path))
+        path = settings.encodings_path.joinpath("viz_config.lp")
+        ctl.load(str(path))
+        path = settings.encodings_path.joinpath("ooasp_aux_kb.lp")
+        ctl.load(str(path))
 
         ctl.add("base",[],self.fb.asp_str())
         ctl.add("base",[],self.kb.fb.asp_str())
         ctl.ground([("base", [])],ClingraphContext())
         ctl.solve(on_model=lambda m: fbs.append(Factbase.from_model(m,default_graph="config")))
         graphs = compute_graphs(fbs[0])
+        # g = unflatten(str(graphs['config']),chain=2)
+        # graphs['config']=Source(g)
```

Comment:
Please don't include commented-out code, because for readers of the code it will be difficult to understand what it means and it will also be hard to maintain.